sdk/servicebus/service-bus/browser/NOTICE.txt

```diff
@@ -0,0 +1,85 @@
+NOTICES AND INFORMATION
```

Comment:
Copy ThirdPartyNotices.txt from other libraries, not from the open source guidelines for now. I believe our other packages have the most up to date version. The note about 3rdpartysource.microsoft.com is inaccurate anyway.

Reply:
Addressed comments and updated to use the format / contents used in other 'ThirdPartyNotices.txt' files instead.